plugins/polio/js/src/pages/Calendar.js

```diff
@@ -97,7 +107,7 @@ const Calendar = ({ params }) => {
         const element = document.getElementById('pdf');
         const options = {
             filename: 'calendar.pdf',
-            excludeTagNames: 'button',
+            excludeTagNames: ['button', 'a'],
```

Comment:
by doing this you can probably remove other infos than the export button, maybe try to exclude a class instead. 
Ask @codeKameleon if he can help you on this